scala/build/Dockerfile.maintenance

```
@@ -0,0 +1,263 @@
+# Install AWS CLI
+# RUN curl -O https://bootstrap.pypa.io/get-pip.py \
+# 	&& python3 get-pip.py --user \
+# 	&& pip3 install awscli --upgrade --user \
+# 	&& rm get-pip.py
+RUN pip install awscli --upgrade --user
+
+# Configure aws
+RUN aws configure set default.region ap-northeast-1
+
+# Install JQ
+RUN mkdir /root/bin
```

Hadolint

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)